docs/users/forwarding.md

```diff
@@ -0,0 +1,61 @@
+# Configuring player information forwarding
+
+Velocity supports forwarding information about your players to your servers, such as IP addresses, UUIDs and skins.
```

Comment:
`Velocity supports forwarding player information such as IP addresses, UUIDs and skins to your servers. This is also known as IP forwarding.`

Honestly, this needs some serious revision (why should people configure modern forwarding?)

Reply:
Would it be better to just leave that to line 7? As it explicitly talks abot modern forwarding and is directly below?